presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -479,6 +479,30 @@ Alter table operations are supported in the connector::
 
      ALTER TABLE iceberg.web.page_views DROP COLUMN location;
 
+TRUNCATE
+^^^^^^^^
+
+The iceberg connector can delete all of the data data from tables without
```

Comment:
```suggestion
The iceberg connector can delete all of the data from tables without
```

Reply:
Fixed, thanks for pointing out the typo!